docs/CHANGELOG.rst

```diff
@@ -6,6 +6,15 @@ All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 
+==========
+Unreleased
+==========
+
+Added
+-----
+- Github actions workflow file
```

Comment:
This is not a user-facing change, so you don't need to add a changelog.

Reply:
Great.